lib/internal/loader/Loader.js

```diff
@@ -25,6 +27,13 @@ function getBase() {
   }
 }
 
+function normalizeReferrerURL(referrer) {
+  if (path.isAbsolute(referrer)) {
+    return getURLFromFilePath(referrer);
+  }
+  return referrer;
```

Comment:
Is this check needed because the `referrer` is a file path for scripts and a URL for modules?

Reply:
Yep, exactly.